### PR TITLE
Adding data-sources.md to documentation

### PR DESCRIPTION
**Pull request by jimenasalinas**
Added an .md file documenting our project's main data sources for GTFS, demographics, and ridership data. The documentation covers details for the 3 cities covered in our project, and includes information about important variables or limitations related to the different data sets used. 

- The documentation has been shared with the team via Box prior to this PR. 

The automatic tests will fail since there is no testing associated with the Markdown update.

### CODE REVIEW

**Review by JPMartinezClaeys — COMMENTED** [review]
@benjaleivas I just wanted to confirm which variables we are using for the Census API to be consistent between the models and this document.

**Comment by meganhmoore** [comment]
why are the tests still failing 😭 how can there not be a pyproject.toml

**Comment by jimenasalinas** [comment]
> @benjaleivas I just wanted to confirm which variables we are using for the Census API to be consistent between the models and this document.

I just added the variables that were missing to the list, we could add the codes as well if we have them

**Comment by jimenasalinas** [comment]
> why are the tests still failing 😭 how can there not be a pyproject.toml

looks like this is solved now :)

**Review by mbjackson-capp — APPROVED** [review]
Might be worth mentioning for Chicago under "GTFS Data Overview" that for CTA, train and bus data are together in each file -- I don't think we have a reason to separate them, but in case it becomes relevant later

File: documentation/data-sources.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+    - Portland (Multnomah County, FIPS: 41051)
+
+#### Data Sources and Geography Levels
+
+The demographic data ingestion scripts utilize data from the following sources:
+
+- Census 2020: Provides data at the state, county, and census tract levels.
+
+- ACS 2022 5-Year Estimates: Provodes data at the state, county, census tract, and block group levels.
```
Comment (JPMartinezClaeys) [inline documentation/data-sources.md:56]: Change provodes to provides

File: documentation/data-sources.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+- Median Household Income (2022 Inflation Adjusted) (ID: B29004_001E): Represents the median household income for the specified area.
+
+- Bachelor's Degree (ID: B15003_022E): Represents the count of individuals holding a bachelor's degree.
+
+- Age: Represents demographic breakdowns by age groups.
+
+- Disability: Represents data related to individuals with disabilities.
+
+- Toatl People, Transportation for work:	Total number of people considered for mean of transportation to work question
```
Comment (JPMartinezClaeys) [inline documentation/data-sources.md:74]: Typo, change to total

**Review by JPMartinezClaeys — CHANGES_REQUESTED** [review]
I found 2 small typos. Feel free to merge after correcting those 2.

**Pull request merged**